integration-tests/cli/csv-parser-price.it.js

```diff
@@ -119,6 +116,26 @@ describe('CSV and CLI Tests', () => {
         },
       )
     })
+
+    test('should log errors to stderr and log file', (done) => {
```

Comment:
In the csv-parser-order package we decided to log all errors to stderr even if we use stdout for output and all messages are being logged to a log file. With this even if there are some errors, a user will see what happened without any need to open a log file.